velero-plugins/main.go

```diff
@@ -30,6 +31,7 @@ func main() {
 		RegisterRestoreItemAction("is-restore-plugin", newImageStreamRestorePlugin).
 		RegisterBackupItemAction("route-backup-plugin", newRouteBackupPlugin).
 		RegisterRestoreItemAction("route-restore-plugin", newRouteRestorePlugin).
+		RegisterRestoreItemAction("build-restore-plugin", newBuildRestorePlugin).
```

Comment:
You're defining the build backup plugin in `build/backup.go`, but you're not registering it here.

Reply:
Yeah right now it's not doing anything so I didn't want to include it. Just figured something *might* go there later so wanted the skeleton code done

Reply:
I wonder what we should do about this sort of thing in general. I'm assuming for release, we wouldn't want to include the backup plugin if it's not being used, but if there's a good chance we'll need some functionality here soon, it makes sense to have it. Perhaps comment in the src file documenting that it's not currently registered would be good, otherwise someone adding code to it might be confused when it's not having an effect.

Reply:
Removed the backup file for now